### PR TITLE
Fix UISeedPhraseTextView behaviour.

### PR DESCRIPTION
**Pull request by SAnatoliiS**

### CODE REVIEW

File: casts/addressText/src/UIAddressTextView.tsx [hunk 1]
```diff
@@ -31,12 +31,16 @@ type UIAddressTextViewProps = UIMaterialTextViewProps & {
     };
 };
 
-function useAddressTextView(ref: React.Ref<TextInput> | null, props: UIAddressTextViewProps) {
+function useAddressTextView(props: UIAddressTextViewProps) {
+    /**
+     * ref is passed as null because types of ref are incompatible
+     * and we don't need the clear method
```
Comment (savelichalex) [inline casts/addressText/src/UIAddressTextView.tsx:37]: Are you sure we don't need it? 🤔

File: kit/inputs/src/UIMaterialTextView/hooks/useImperativeChange.ts [hunk 2]
```diff
@@ -35,16 +30,9 @@ export function useImperativeChange(
                 });
             }
 
-            if (multiline) {
-                if (Platform.OS === 'web') {
-                    const elem = ref.current as unknown as HTMLTextAreaElement;
-                    calculateWebInputHeight(elem);
```
Comment (savelichalex) [inline kit/inputs/src/UIMaterialTextView/hooks/useImperativeChange.ts:41]: So, it doesn't work anymore or why it's dropped?

File: casts/addressText/src/UIAddressTextView.tsx (hunk 1 above)
Comment (SAnatoliiS) [reply casts/addressText/src/UIAddressTextView.tsx:37]: clear method returns from useUITextViewValue but it doesn't take here.

File: kit/inputs/src/UIMaterialTextView/hooks/useImperativeChange.ts (hunk 2 above)
Comment (SAnatoliiS) [reply kit/inputs/src/UIMaterialTextView/hooks/useImperativeChange.ts:41]: It was duplicate logic. It works in useAutogrow.web.ts.

**Review by savelichalex — APPROVED** [review]

**Review by sertony — APPROVED** [review]

**Pull request merged**